src/main/resources/application.yml

```diff
@@ -40,12 +38,6 @@ server:
     remote-ip-header: x-forwarded-for
     protocol_header: x-forwarded-proto
 
-logging:
```

Comment:
was this just not needed?

Reply:
agreed, we already have it in logback-spring.xml so having it in two places was really confusing and caught me out in token verification service